File: CLASS 1/8958 OX퀴즈.cs
using System;

namespace ConsoleApp1
{
    public class Program
    {
        static void Main()
        {
            int testcaseN = int.Parse(Console.ReadLine());
            int sum = 0;
            int num;

            while (testcaseN > 0)
            {
                num = 1;

                string str = Console.ReadLine();
                foreach (char str2 in str)
                {
                    if (str2 == 'O')
                    {
                        sum += num;
                        num++;
                    }
                    else
                    {
                        num = 1;
                    }
                }
                Console.WriteLine(sum);
                sum = 0;

                testcaseN--;
            }
        }
    }
}

File: CLASS 1/8958 OX퀴즈.c
#define _CRT_SECURE_NO_WARNINGS // scanf사용시 에러무시
#include <stdio.h>

int main(void)
{
    int testcaseN;
    int sum = 0;
    char c;

    scanf("%d", &testcaseN);
    c = getchar();

    int num;
    while (testcaseN)
    {
        num = 1;
        while (1)
        {
            c = getchar();
            if (c == 'O')
            {
                sum += num;
                num++;
            }
            else if(c == 'X')
            {
                num = 1;
            }
            else
            {
                printf("%d\n", sum);
                sum = 0;
                break;
            }
        }
        testcaseN--;
    }

    return (0);
}
